### PR TITLE
Publish ocis release notes with content

### PR DESCRIPTION
**Pull request by mmattel**
This PR publishes the Infinite Scale Releases/Release Notes documents.

* The content of the release notes have been derived from [docs-only] Infinite Scale 2.0.0 beta1 Release Notes
* **NEW** 😄 
We do not need to manually maintain the table of contents in the ocis _release notes_ anymore. This is now automated
* The content of _releases_ has been adopted to how ocis docs are setup

@pmaier1 this is the followup PR to bring content into the prepared notes and publishes them. Created so that you have minimal efforts. Can you pls check and approve if ok.

### CODE REVIEW

File: modules/ROOT/pages/ocis_releases.adoc [hunk 1]
```diff
@@ -1,15 +1,15 @@
 = Infinite Scale Server Releases
 :toc: right
-:toclevels: 3
+:toclevels: 1
 
-:description: This is the introduction text to be defined, max 160 chars if possible. 
+:description: ownCloud provides in-depth documentation for Infinite Scale. Functionalities added or decommissioned dependent to a particular version are referenced in the documentation directly.
```
Comment (phil-davis) [inline modules/ROOT/pages/ocis_releases.adoc:5]:
```suggestion
:description: ownCloud provides in-depth documentation for Infinite Scale. Functionalities added or decommissioned dependent on a particular version are referenced in the documentation directly.
```

**Review by phil-davis — COMMENTED** [review]
Text generally looks good.

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 2]
```diff
@@ -1,16 +1,64 @@
...
+
+You can read more about the Infinite Scale beta phase on https://owncloud.com
+
+- https://owncloud.com/news/infinite-scale-beta-available/[Welcome to the Infinite Scale Beta]
+- https://owncloud.com/infinite-scale-beta[Infinite Scale Beta Page]
+- https://owncloud.com/news/infinite-scale-beta/[Infinite Scale Beta is coming]
+
+The first beta release of Infinite Scale 2.0.0 (beta1) introduces the `File Search` feature and completely replaces the integrated user management with a lightweight LDAP server that is shipped out-of-the-box (LibreIDM). ownCloud Web introduces a new, feature-rich upload manager based on uppy.io and comes with a couple of design and usability round-offs.
+
+The most prominent changes in **ownCloud Infinite Scale 2.0.0 beta1** and ownCloud Web 5.5.0 comprise:
```
Comment (EParzefall) [inline modules/ROOT/pages/ocis_release_notes.adoc:29]:
```suggestion
The most prominent changes in **Infinite Scale 2.0.0 beta1** and ownCloud Web 5.5.0 comprise:
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 3]
```diff
@@ -1,16 +1,64 @@
...
+
+* All `Breaking changes` of Infinite Scale 1.20.0 Technology Preview have been fixed. Especially the https://github.com/cs3org/wopiserver[WOPI Server extension] is compatible with Infinite Scale 2.0.0 again.
```
Comment (EParzefall) [inline modules/ROOT/pages/ocis_release_notes.adoc:31]:
```suggestion
* All `breaking changes` of Infinite Scale 1.20.0 Technology Preview have been fixed. Especially the https://github.com/cs3org/wopiserver[WOPI Server extension] is compatible with Infinite Scale 2.0.0 again.
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 4]
```diff
@@ -1,16 +1,64 @@
...
+
+* The existing integrated user management has been replaced by LibreIDM which is a lightweight LDAP server being developed by ownCloud and the LibreGraph community (https://github.com/libregraph/idm). LibreIDM comes with a MS Graph-based API and integrates a user interface for user & group management in ownCloud Web. https://github.com/owncloud/ocis/pull/3331[ocis#3331], https://github.com/owncloud/web/issues/6673[web#6673]
+
+* For security reasons, Infinite Scale will not start anymore without specifying certain secrets. A new command `ocis init` has been added to automatically prepare a new Infinite Scale installation by generating secure default secrets. https://github.com/owncloud/ocis/pull/3551[ocis#3551]
+
+* The `Shares` folder has been moved into a dedicated entry point. It is now available in the left sidebar as `Shares` and will gather all incoming and outgoing shares of the user. With this, the transition to a clear separation of personal, shared and project files is complete. https://github.com/owncloud/web/issues/6448[web#6448]
+
+* Spaces cover images will now be rendered with an appropriate aspect ratio (16:9). https://github.com/owncloud/web/pull/6829[web#6829]
```
Comment (EParzefall) [inline modules/ROOT/pages/ocis_release_notes.adoc:43]:
```suggestion
* Cover images for spaces will now be rendered with an appropriate aspect ratio (16:9). https://github.com/owncloud/web/pull/6829[web#6829]
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 5]
```diff
@@ -1,16 +1,64 @@
...
+
+* When un-sharing a file/folder, a confirmation dialog will now appear https://github.com/owncloud/web/pull/6795[web#6795]
```
Comment (EParzefall) [inline modules/ROOT/pages/ocis_release_notes.adoc:45]:
```suggestion
* When unsharing a file or folder, a confirmation dialog will now appear. https://github.com/owncloud/web/pull/6795[web#6795]
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 6]
```diff
@@ -1,16 +1,64 @@
...
+
+You can also read the full https://github.com/owncloud/ocis/releases/tag/v2.0.0[ownCloud Infinite Scale changelog] and https://github.com/owncloud/web/releases/tag/v5.5.0[ownCloud Web changelog] for further details on what has changed.
+
+=== Known issues
 
-=== Migrations
+This section will be updated if known issues are discovered.
 
-=== Other Notable Changes
+* Copy & Move in ownCloud Web does not work outside of personal spaces https://github.com/owncloud/web/issues/6892[web#6892]
```
Comment (EParzefall) [inline modules/ROOT/pages/ocis_release_notes.adoc:59]:
```suggestion
* Copy & Move in ownCloud Web does not work outside of personal spaces. https://github.com/owncloud/web/issues/6892[web#6892]
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 7]
```diff
@@ -1,16 +1,64 @@
...
+* Single file user shares have no download action and thumbnail creation does not work for the recipient
```
Comment (EParzefall) [inline modules/ROOT/pages/ocis_release_notes.adoc:60]:
```suggestion
* Single file user shares have no download action and thumbnail creation does not work for the recipient.
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 8]
```diff
@@ -1,16 +1,64 @@
...
+* Received shares cannot be renamed https://github.com/owncloud/ocis/issues/3721[ocis#3721]
```
Comment (EParzefall) [inline modules/ROOT/pages/ocis_release_notes.adoc:61]:
```suggestion
* Received shares cannot be renamed. https://github.com/owncloud/ocis/issues/3721[ocis#3721]
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 9]
```diff
@@ -1,16 +1,64 @@
...
+* The personal user quota is currently hardcoded https://github.com/owncloud/ocis/issues/3748[ocis#3748]
```
Comment (EParzefall) [inline modules/ROOT/pages/ocis_release_notes.adoc:62]:
```suggestion
* The personal user quota is currently hardcoded. https://github.com/owncloud/ocis/issues/3748[ocis#3748]
```

**Review by EParzefall — APPROVED** [review]

**Comment by mmattel** [comment]
Beside the changes above, I will add after a discussion with @pmaier1 more release changes

**Pull request merged**